sdks/python/apache_beam/utils/shared.py

```diff
@@ -55,18 +50,17 @@ def process(self, element):
       yield self._giant_list[element]
 
   p = beam.Pipeline()
-  shared_handle = shared.Shared()
   (p | beam.Create([2, 4, 6, 8])
-     | beam.ParDo(GetNthStringFn(shared_handle)))
+     | beam.ParDo(GetNthStringFn()))
 
 
 Real-world uses will typically involve using a side-input to a DoFn to
```

Comment:
This documentation is still confusing to me.

Reply:
I agree there's still room for improvement.